app/Lib/Rscripts/analyse.R

```diff
@@ -94,9 +136,14 @@ Analyse <- function(key, input.answers, number.answeroptions) {
       percentage.answer.options <- 0
       corrected.item.tot.cor.answ.option <- 0
     }
-    
-    # Computes Cronbach's Alpha
-    cronbach <- round(cronbach(input.correct)$alpha, digits = 3)
+
+    # Computes Cronbach's Alpha for overall test
+    # If there is no variance in the total score returns -100
+    if(!all(rowSums(input.correct) == rowSums(input.correct)[1])){
+    	cronbach <- round(cronbach(input.correct)$alpha, digits = 3)
+    } else {
+    	cronbach <- -100
```

Comment:
@SjoerdGlaser Is it possible to make this a value in the range from -9.999 until 9.999? In the current database schema it is not possible to save a value out of this range.